config/routes.rb

```diff
@@ -279,6 +281,7 @@
   get "manufacturers_tsv", to: "manufacturers#tsv"
 
   resource :integrations, only: [:create]
+  get "/auth/twitter/callback", to: "admin/twitter_accounts#create"
```

Comment:
I like moving the integration to admin rather than handling it in the integrations controller, good call.